File: src/test/java/io/holunda/connector/LocalConnectorRuntime.java
package io.holunda.connector;

import io.camunda.connector.runtime.ConnectorRuntimeApplication;
import org.springframework.boot.SpringApplication;

public class LocalConnectorRuntime {

  public static void main(String[] args) {
    SpringApplication.run(ConnectorRuntimeApplication.class, args);
  }
}

File: src/test/kotlin/io/holunda/connector/LocalConnectorRuntime.kt
package io.holunda.connector

import io.camunda.connector.runtime.ConnectorRuntimeApplication
import org.springframework.boot.SpringApplication

object LocalConnectorRuntime {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(ConnectorRuntimeApplication::class.java, *args)
    }
}
